models.py

```diff
@@ -253,7 +256,7 @@ def callbacks_unet(callbacks_config):
                    ])
 
 
-def multiclass_weighted_segmentation_loss(output, target, w0, sigma):
+def multiclass_weighted_segmentation_loss(output, target, w0, sigma, C):
```

Comment:
@apyskir this is in fact multiclass_weighted_cross_entropy. The whole idea of segmentation_loss (which may actually be entirely flawed when you can simply sum stuff) was to have one function that handles both bce and dice. So I would change this name

Reply:
@jakubczakon OK, fine.